docs/source/array_stats_only.ipynb

```diff
@@ -0,0 +1,252 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2aded49a-be14-4e0e-95e1-12474dec548b",
+   "metadata": {},
+   "source": [
+    "# Using `arviz_stats` array interface\n",
+    "\n",
+    "This tutorial will cover how to use the `arviz_stats` array interface for diagnosing and summarizing Bayesian modeling results stored as\n",
+    "NumPy arrays. It is aimed at advanced users and developers of other libraries such as PPLs who want to incorporate sampling diagnosting into their library.\n",
+    "\n",
+    "## What is the \"array interface\"?\n",
+    "\n",
+    "The array interface is the base building block on top of which everything within `arviz_stats` is built, and is always available.\n",
+    "When you install `arviz_stats` as `pip install arviz_stats` (instead of the recommended way shown in {ref}`installation`) you get\n",
+    "a minimal package installed with NumPy and SciPy as the only dependencies and `array_stats` as the way to interface with the functions of the library.\n",
+    "\n",
+    "As there is no dependency on `arviz_base` defaults are either hardcoded or not set, making some arguments that are optional when using the\n",
+    "top level functions or xarray interfaces required ones. In addition, as it is one of the building blocks of the DataArray interface\n",
+    "which uses {func}`xarray.apply_ufunc` the default axis to be reduced are the last one(s). \n",
```

Comment:
I found this paragraph potentially confusing. I think it should start saying something like "As the array interface does not depends on `arviz_base`..." and then reword the last sentence, no sure how.... maybe " In addition, because the array interface is a building block of the DataArray interface, which uses {func}xarray.apply_ufunc, the default reduction is performed over the last axis or axes."

Reply:
I'll move it later on to the example with psislw to explain why the order of the dimensions changes it is not that critical either as most common functions reduce these dimensions so the dimensions being moved before reduction or not internally is all the same to users.